### PR TITLE
feat: LEAP-1097: Show Review buttons in Quick View

### PR DESCRIPTION
**Pull request by hlomzik**
If that's the task of another person we will show Accept/Reject. Part of "Reviewer Flow" experiment, hidden behind FF.

- [x] Check if user is of Manager role
- [x] Check if we are in Quick View

### PR fulfills these requirements
- [ ] Tests for the changes have been added/updated
- [ ] Docs have been added/updated
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance

### Describe the reason for change
_(if it's not clear from main description — otherwise remove this section)_

### What feature flags were used to cover this change?
`fflag_feat_all_leap_1081_reviewer_flow_updates`

### This change affects (describe how if yes)
- [ ] Performance
- [ ] Security
- [x] UX

### Does this PR introduce a breaking change?
- [x] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
- [ ] e2e (codecept)
- [ ] integration (cypress)
- [ ] unit (jest)

### CODE REVIEW

File: web/libs/editor/src/stores/Annotation/Annotation.js [hunk 1]
```diff
@@ -339,6 +340,23 @@ export const Annotation = types
...
+    // experiment to display review buttons in Quick View
+    get canBeReviewed() {
+      const store = self.store;
+
+      return isFF(FF_REVIEWER_FLOW)
+        // not a current user — we can only review others' annotations
+        && self.user?.email
+        && store.user?.email !== self.user?.email
+        // we have this only in LSE
+        && store.hasInterface("annotations:history")
```
Comment (hlomzik) [inline web/libs/editor/src/stores/Annotation/Annotation.js]: most probably it's also added as part of default interfaces for LSO, so we should find another way to distinguish LSO and LSE. @Gondragos do you have any ideas?

Comment (Gondragos) [reply web/libs/editor/src/stores/Annotation/Annotation.js]: What about just defining a new interface for exactly that purpouse?

Comment (Gondragos) [reply web/libs/editor/src/stores/Annotation/Annotation.js]: Or.... there is an option
```js
getEnv(self).events.hasEvent("acceptAnnotation")
```

**Review by Gondragos — APPROVED** [review]

**Comment by hlomzik** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/9418257022)
> Successfully merged:  7 files changed, 109 insertions(+), 150 deletions(-)

**Comment by hlomzik** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/9447685645)
> Successfully merged:  34 files changed, 1598 insertions(+), 78 deletions(-)

**Pull request merged**